ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -45,10 +46,13 @@
   private boolean isOpen;
   private final FrontendConfig frontendConfig;
   private final FrontendMetrics frontendMetrics;
+  private final AuthorizationService authorizationService;
```

Comment:
Hmm I see. We will have to figure this out. Ideally, AclService will still be used within SecurityService, since it is security related, but it also needs some entity to find the ACLed resource. It is possible for this to happen in the IdConverter implementation, but it seems a bit out of place for this to occur there.